_source/_posts/2022-01-26-webassembly-on-kubernetes-with-rust.md

```diff
@@ -0,0 +1,365 @@
+---
+layout: blog_post
+title: "Containerless! How to Run WebAssembly Workloads on Kubernetes with Rust"
+author: deepu-sasidharan
+by: advocate
+communities: [devops, rust]
+description: "Learn how to run secure WebAssembly workloads written with Rust on Kubernetes with Krustlet."
+tags: [rust, webassembly, kubernetes, security]
+tweets:
+  - "Do you want to go Containerless on Kubernetes? Learn how to run WebAssembly workloads written in Rust on Kubernetes. #rust #webassembly #kubernetes #security"
+  - "Learn how to run secure WebAssembly workloads written with Rust on Kubernetes with Krustlet. #rust #webassembly #kubernetes #security"
+  - "Is WebAssembly the future of Kubernetes? Learn all about Wasm on Kubernetes with Rust. #rust #webassembly #kubernetes #security"
+image: blog/webassembly-on-kubernetes-with-rust/rust_wasm_k8s.jpg
+type: awareness
+---
+
+WebAssembly (Wasm) is one of the most exciting and underestimated software technologies invented in recent times. It's a binary instruction format for a stack-based virtual machine. It aims to execute at native speeds with a memory-safe and secure sandbox. Its portable, cross-platform, and language-agnostic. It's designed as a compilation target for languages. Though originally part of the open web platform, it has found use cases beyond the web. WebAssembly is now used in browsers, Node.js, Deno, Kubernetes, and IoT platforms.
```

Comment:
```suggestion
WebAssembly (Wasm) is one of the most exciting and underestimated software technologies invented in recent times. It's a binary instruction format for a stack-based virtual machine that aims to execute at native speeds with a memory-safe and secure sandbox. Wasm is portable, cross-platform, and language-agnostic—designed as a compilation target for languages. Though originally part of the open web platform, it has found use cases beyond the web. WebAssembly is now used in browsers, Node.js, Deno, Kubernetes, and IoT platforms.
```
Great intro paragraph - these suggestions break up the repetitive use of "It is..." to start the sentences for ease of reading. Also for variation of sentence length.